packages/mds-policy-service/service/provider.ts

```diff
@@ -108,6 +108,13 @@ export const PolicyServiceProvider: ServiceProvider<PolicyService, PolicyService
         policy_id,
         policy_metadata: { intent_type: intent_draft.intent_type }
       })
-      return PolicyRepository.publishPolicy(policy_id)
+      const published_date = now()
+      return PolicyRepository.publishPolicy(policy_id, published_date, {
+        beforeCommit: process.env.KAFKA_HOST
+          ? async policy => {
+              return await PolicyStreamKafka.write({ ...policy, published_date })
```

Comment:
`published_date` actually does _not_ need to be specifically added here with a spread, as the `policy` object that is supplied when this function is called within `PolicyRepository.publishPolicy` already includes the `published_date`.

See packages/mds-policy-service/repository/index.ts::423-439
```suggestion
      return PolicyRepository.publishPolicy(policy_id, now(), {
        beforeCommit: process.env.KAFKA_HOST
          ? async policy => {
              return await PolicyStreamKafka.write(policy)
```

Reply:
^^ this is the thing I'm requesting changes for, everything else is just nits